src/Libraries/CoreNodeModels/ColorRange.cs

```diff
@@ -145,12 +180,44 @@ public ColorRange1D ComputeColorRange(EngineController engine)
             else
             {
                 parameters = CreateParametersForColors(colors);
+
+                // Create an AssociativeNode for the default indices
+                InPorts[1].DefaultValue = DefaultIndicesNode;
+                InPorts[1].UsingDefaultValue = true;
             }
 
             return ColorRange1D.ByColorsAndParameters(colors, parameters);
         }
 
-        private static List<double> CreateParametersForColors(List<Color> colors)
+        private AssociativeNode CreateDefaultColorsNode(IEnumerable<Color> defaultColors)
+        {
+            return AstFactory.BuildExprList(
+                    defaultColors.Select(color =>
+                        AstFactory.BuildFunctionCall(
+                            new Func<int, int, int, int, Color>(DSCore.Color.ByARGB),
+                            new List<AssociativeNode>
+                            {
```

Comment:
Missed the Opacity, `A` here resulting in that issue. Fixing in 3.6.1 with a PR.